libethdrivers/plat_include/imx6/ethdrivers/plat/eth_plat.h

```diff
@@ -14,11 +14,11 @@
 
 struct enet *get_enet_from_driver(struct eth_driver *driver);
 
-enum {
+enum nic_config_flags_t {
```

Comment:
why is this needed?

Reply:
This changes nic_config_flags_t from a global with an enum type to a enum type declaration.  Recent compiler versions (gcc-10) no longer place the global definitions in a COM section and this leads to linker errors. Alternatively this could be changed a different way to add a typedef at the start.

Reply:
I think the typedef is what we need here, this was never supposed to be a global variable.

Reply:
issue looks similar to https://github.com/seL4/camkes-vm/pull/13